events/handler_test.go

```diff
@@ -87,8 +93,13 @@ func TestCheckIfIsSubcontainer(t *testing.T) {
 			myRequest, subContainerEvent)
 	}
 
+	rootRequest.IncludeSubcontainers = true
 	myRequest.IncludeSubcontainers = true
 
+	if !checkIfIsSubcontainer(rootRequest, sameContainerEvent) {
+		t.Errorf("should have found %v and %v had the same container",
+			rootRequest.ContainerName, sameContainerEvent.ContainerName)
+	}
 	if !checkIfIsSubcontainer(myRequest, sameContainerEvent) {
```

Comment:
I *think* this one is fine, we check against subcontainer below it. We check:
- `/root` subcontainerOf `/root` == `true`
- `/root/sub` subcontainerOf `/root` == `true`
- `/root-different` subcontainerOf `/root` == `false`